### PR TITLE
Update licenses-assign.md

### PR DESCRIPTION
**Pull request by msbemba**
added more information about device licenses

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3748

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 PLease copy review

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: memdocs/intune/fundamentals/licenses-assign.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ See this [overview of School Data Sync](https://support.office.com/article/Overv
 ## How user and device licenses affect access to services
 
 - Each **user** that you assign a user software license to may access and use the online services and related software (including System Center software) to manage applications and up to 15 MDM devices.
-- You can purchase licenses for any devices separately from user licenses. Device licenses do not need to be assigned to the devices. Each device that accesses and uses the online services and related software (including System Center software) must have a device license.
+- You can purchase licenses for any devices separately from user licenses. Device licenses do not need to be assigned to the devices. Each device that accesses and uses the online services and related software (including System Center software) must have a device license available in the M365 tenant.
```
Comment (JohanFreelancer9) [inline memdocs/intune/fundamentals/licenses-assign.md:71]:
```suggestion
- You can purchase licenses for any devices separately from user licenses. Device licenses do not need to be assigned to the devices. Each device that accesses and uses the online services and related software (including System Center software) must have a device license available in the Microsoft 365 tenant.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**